src/Nethermind/Nethermind.Flashbots/Handlers/ValidateBuilderSubmissionHandler.cs

```diff
@@ -174,7 +174,7 @@ private bool ValidateBlobsBundle(Transaction[] transactions, BlobsBundleV1 blobs
             return false;
         }
 
-        if (!IBlobProofsManager.For(ProofVersion.V1).ValidateProofs(new ShardBlobNetworkWrapper(blobsBundle.Blobs, blobsBundle.Commitments, blobsBundle.Proofs, ProofVersion.V1)))
+        if (!IBlobProofsManager.For(releaseSpec.BlobProofVersion).ValidateProofs(new ShardBlobNetworkWrapper(blobsBundle.Blobs, blobsBundle.Commitments, blobsBundle.Proofs, ProofVersion.V1)))
```

Comment:
Shouldn't it also be `releaseSpec.BlobProofVersion` instead of `ProofVersion.V1` as an argument for `new ShardBlobNetworkWrapper`?

In current version it will be always invalid before Fusaka, as far as I understand

Reply:
Indeed, nice catch!